.github/workflows/linters.yaml

```diff
@@ -33,7 +33,7 @@ jobs:
 
     - uses: actions/setup-python@v4
       with:
-        python-version: 3.9
+        python-version: "3.10"
```

Comment:
@RMeli sorry I don't fully understand if you mean this or the actual darker action version number

Reply:
Ah thanks! Totally forgot that was even a thing 😅

Reply:
done!